### PR TITLE
PEDS-939 - update portal with ga4

### PR DESCRIPTION
**Pull request by skyzhangty**
Ticket [PEDS-939](https://pcdc.atlassian.net/browse/PEDS-939)

Updated with GA4. Need to update the new google tracking id in the configuration file repo

[PEDS-939]: https://pcdc.atlassian.net/browse/PEDS-939?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/index.jsx [hunk 1]
```diff
@@ -12,21 +12,15 @@ import {
...
 import { basename } from './localconf';
 import App from './App';
 import '@fontsource/raleway';
 import './base.css';
 import './icon.css';
 
-// Google Analytics
-ReactGA.initialize(gaTracking);
-ReactGA.pageview(window.location.pathname + window.location.search);
-
```
Comment (grugna) [inline src/index.jsx:29]: @skyzhangty does this need to be replaced with ga4 or it is superfluous and removed on purpose?

**Comment by grugna** [comment]
Do these changes need to be coordinated with https://pcdc.atlassian.net/browse/PEDS-937 and upgrade the version on the console, or they are retro compatible and if we merge this branch without updating right away the google analytics console they will still work?

**Comment by skyzhangty** [comment]
@grugna In non-prod env, we don't use google analytics (gaTrackingId is undefined), so it should be fine to merge and deploy to non-prod env. In prod, we need to update the config file https://github.com/chicagopcdc/configuration-files/blob/pcdc_dev/cloud/portal.pedscommons.org/portal/gitops.json and set ```gaTrackingId``` to ```G-15T74ER5GN``` as  https://pcdc.atlassian.net/browse/PEDS-937 indicated

File: src/index.jsx (hunk 1 above)
Comment (skyzhangty) [reply src/index.jsx:29]: This is removed on purpose. We have a ```Layout``` component that's the first component in router and is always going to the rendered. In this ```Layout``` component, we set up Google Analytics. I just think there's no need to set it up again in the index.jsx again. I tested it even when I go to a page directly (not via index page) such as the explorer page, the GA related scripts are added to the HTML.

**Pull request merged**